rfcs/2021-28-07-motion-design-core-actions.md

```diff
@@ -0,0 +1,176 @@
+
+### 2. **Move**
+* **Anchor points(`I` & `F`):** Initial & Final anchor-point coordinates; `(x1, y1)(x2, y2)`.
+* **Pathway:** Control points (which decide curve of the line) [path](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths) i.e., how the element moves from `position1` to `position2` representing the pathways; depending on the surroundings or the `z-axis`.
+
+
+### Pathway Types:
+| Type | Description | Example |
+|---|---|---|
+| **Type 1:** <br><br> <img src="images/motion/core-actions/image6.gif" alt="core-action-scale-expand" width="200px"> | **Elements in the same layer (`z-space`)** <br>  Use type 1 pathway, it’s straight-forward and avoids unnecessary distraction.| <img src="images/motion/core-actions/image8.png" alt="core-action-scale-expand" width="200px"> |
+| **Type 2:**  <br><br> <img src="images/motion/core-actions/image17.gif" alt="core-action-scale-expand" width="200px">| **No other element in the same layers** <br> Use type-2 pathway, it makes space for the movement<br>**Ex:** Check out Spotify play icons and top-bar icons.  | <img src="images/motion/core-actions/image7.gif" alt="core-action-scale-zoom" width="200px"> |
+
+### 3. **Fade**
+* **Opacity:** Percentage(`%`) change in the `opacity`.
+
+### Types:
+| Type | Description | Example |
+|---|---|---|
+| **Fade In** | Opacity value moves from a lower to higher. <br>**Ex:** Appearing of a card on a page.<br>**Unit:** `%` | <img src="images/motion/core-actions/image9.gif" alt="core-action-fade-in" width="200px"> |
+| **Fade Out** | Opacity value moves from a higher to lower.<br>**Ex:** Disappearing of a card from a page.<br>**Unit:** `%` | <img src="images/motion/core-actions/image9.gif" alt="core-action-fade-out" width="200px"> |
+
+### 4. **Fill**
+* **Color tokens:** Predefined Color tokens within the design system and changing or adding different tokens for diffrent state leads to change in the color value as well.
+
+### Types:
+| Type | Description | Example |
+|---|---|---|
+| **Color token change** | When there is a change in state of a UI element, use the fill effect. <br>**Ex:** Hovering over a button.<br>**Unit:** token value (`color.btn.primary.hover`) | <img src="images/motion/core-actions/image10.gif" alt="core-action-fill" width="200px"> |
+
+### 5. **Morph**
+> It can be used on mask
+* **Shape:** A graphics element that is defined by some combination of straight lines and curves. Read more on [Shapes](https://www.w3.org/TR/SVG2/shapes.html). Specifically: circle, ellipse, line, path, polygon, polyline and rect.
+
+### Types:
+| Type | Description | Example |
+|---|---|---|
+| **Shape change** | When a shape gets morhed into another. <br>**Ex:** Opening of dropdown menu.<br>**Unit:** - | <img src="images/motion/core-actions/image11.gif" alt="core-action-morph" width="200px"> |
```

Comment:
couldn't quite understand why will we need morp for dropdown? Usually all we add is maybe icons to something when the drop down is clicked

Reply:
https://material.io/design/motion/customization.html#motion-paths
Check out the link, How circle is transformed into Circle. I can give a better example here.